lua/entities/gmod_tardis/modules/sh_teleport.lua

```diff
@@ -363,10 +381,16 @@ else
 			local int = self.metadata.Interior.Sounds.Teleport
 			local pos=net.ReadVector()
 			if LocalPlayer():GetTardisData("exterior")==self then
-				self:EmitSound(ext.mat)
-				self.interior:EmitSound(int.mat or ext.mat)
+				if not self:GetData("demat-fast",false) then
+					self:EmitSound(ext.mat)
+					self.interior:EmitSound(int.mat or ext.mat)
+				end
 			else
-				sound.Play(ext.mat,pos)
+				if self:GetData("demat-fast",false) then
+
```

Comment:
empty block? Invert the if and get rid of the else if there is nothing to be here